samples/bluetooth/periodic_adv_rsp/src/main.c

```diff
@@ -92,9 +92,6 @@ static void response_cb(struct bt_le_ext_adv *adv, struct bt_le_per_adv_response
 	if (buf) {
 		printk("Response: subevent %d, slot %d\n", info->subevent, info->response_slot);
 		bt_data_parse(buf, print_ad_field, NULL);
-	} else {
-		printk("Failed to receive response: subevent %d, slot %d\n", info->subevent,
-		       info->response_slot);
```

Comment:
Since we listen to every response slot even if we dont expect any response this should be ok. Could potentially rephrase the print instead to not indicate an error, but that might get very spammy if the controller generates events for each response slot.

Reply:
yes it looks too spammy.